packages/framework/theme/src/ThemeProvider.tsx

```diff
@@ -16,6 +16,9 @@ export const ThemeProvider: React.FunctionComponent<ThemeProviderProps> = (props
   const [theme, setThemeState] = React.useState(() => themeRef.theme);
 
   React.useEffect(() => {
+    // Result of https://github.com/microsoft/fluentui-react-native/issues/3163
```

Comment:
nit: I don't think we need to reference the issue in the code, we can explain what the intent of the code is directly, or just leave it out. I think it's a bit cumbersome to direct a code reader to an issue to gain context for some code.